main.js

```diff
@@ -455,6 +459,9 @@ document.addEventListener('DOMContentLoaded', function () {
         baseUrl = fileJson.servers[0].url;
```

Comment:
are these lines still valid, or should they be removed?

Reply:
Still valid